lib/cluster-def-gem/cluster_def.gemspec

```diff
@@ -0,0 +1,13 @@
+Gem::Specification.new do |s|
+  s.name        = 'cluster_def'
+  s.version     = '0.1.0'
```

Comment:
Should this come from the cookbook metadata version to ensure it gets consistently versioned?

Reply:
There are no ties between a specific version of the gem or cookbook